app/views/hosts/_list.html.erb

```diff
@@ -17,7 +17,7 @@
     <% hosts.each do |host| %>
       <tr>
         <td class="ca">
-          <%= check_box_tag "host_ids[]", nil, false, :id => "host_ids_#{host.id}", :disabled => !authorized?, :class => 'host_select_boxes', :onclick => 'hostChecked(this)' %>
```

Comment:
No point disabling the checkboxes, this is an expensive check performed for every host (esp. with all the permissions) for very little UX value. If none of the actions are allowed, the drop down will simply not show and this checkbox will have no effect.